packages/xo-web/src/xo-app/pool/tab-network.js

```diff
@@ -101,6 +101,45 @@ class DefaultPif extends BaseComponent {
   }
 }
 
+class Nbd extends Component {
+  NBD_FILTER_OPTIONS = [
+    {
+      label: 'noNbdConnection',
```

Comment:
`labelId`.

Reply:
I don't understand 🤔

Reply:
`labelId: 'noNbdConnection'`. Since it's an ID, not the label itself. Or is `label` required by the select?

Reply:
The ```label``` is used to get the message from ```message.js```.

But it's ok if I also change ```label``` to ```labelId``` in line 123.